.github/workflows/build.yaml

```diff
@@ -0,0 +1,63 @@
+name: Build
```

Comment:
Strange... this shouldn't appear as a change, since it is in master. :thinking:

Reply:
I rebased again, better now?